python/ml_croissant/ml_croissant/_src/datasets_test.py

```diff
@@ -81,12 +74,28 @@
         ],
     ],
 )
-def test_static_analysis(filename, error):
+def test_static_analysis_old(filename, error):
     base_path = epath.Path(__file__).parent / "tests/graphs"
     with pytest.raises(ValidationError, match=error):
         datasets.Dataset(base_path / filename)
 
 
+def get_error_msg(folder):
+    with open(f"{folder}/output.txt", "r") as file:
+        return file.read().strip()
+
+
+# TODO(ccl-core): Progressively move tests from test_static_analysis_old to
```

Comment:
When writing `TODO`, I recently started to refer to GitHub issues, rather than individuals. Like this, everybody can take over a given task/todo. WDYT?

```python
# TODO(https://github.com/mlcommons/croissant/issues/14): ...
```

Reply:
SG, thanks!